### DISCUSSION TITLE
[tab]multiple menu on same page tab

### DISCUSSION THREAD

**Discussion by apple-idn** [fomantic/Fomantic-UI#577] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Help Wanted
how work on same page with multiple menu data-tab
## Problem
`<div class="ui fluid secondary vertical pointing menu">`
`<a class="red item active" data-tab="1">` Menu 1 `</a>`
`<a class="green item" data-tab="2">`Menu 2 `</a>`
`<a class="green item" data-tab="3">`Menu 3` </a>`
`</div>`
 `<div class="ui tab" data-tab="1">`
for menu 1
`</div>`
`<div class="ui tab" data-tab="2;3;">`
for same menu '2 and menu 3'
`</div>`

**Comment by lubber-de** [comment]
- Use `data-tab="2"` also for the "Menu 3" item:
```html
<a class="green item" data-tab="2">Menu 3 </a>
```
- Give the second tab the name 2
```html
<div class="ui tab" data-tab="2">
```

See https://jsfiddle.net/0hqeay3v/

**Comment by ko2in** [comment]
I noticed from your fiddle, if two tab menu point to the same tab, the tab menu that currently clicked failed to mark as active.

**Menu 1, OK**
![Tab1](https://user-images.githubusercontent.com/930315/54818927-d4456a80-4cc8-11e9-8e57-b28ac86d78d1.png)

**Menu 2 and 3 failed**
![Tab2](https://user-images.githubusercontent.com/930315/54818956-e32c1d00-4cc8-11e9-9b9b-28068bbd9c7e.png)

**Comment by apple-idn** [comment]
@lubber-de @ko2in here's my question how to trick different data-tab item with one tab? How This work?

**Comment by ko2in** [comment, marked as answer]
@apple-idn Maybe, this is what you're looking for:
```
<div class="ui fluid secondary vertical pointing menu">
<a class="red item active" data-tab="tab-1"> Menu 1 </a>
<a class="green item" data-tab="global-tab">Menu 2 </a>
<a class="green item" data-tab="global-tab">Menu 3 </a>
</div>
<div class="ui tab" data-tab="tab-1">
for menu 1
</div>
<div class="ui tab" data-tab="global-tab">
for same menu '2 and menu 3'
</div>
```
I'm closing this since this is old issue and the usage question. Please feel free to reopen if it doesn't help.
